src/main/java/com/google/sps/data/Mentee.java

```diff
@@ -1,32 +1,48 @@
 package com.google.sps.data;
 
 import com.google.appengine.api.datastore.Entity;
+import java.util.Collection;
 
 public class Mentee extends UserAccount {
 
   private static final String GOAL = "goal";
   private static final String DESIRED_MEETING_FREQUENCY = "desiredMeetingFrequency";
+  private static final String YESSES = "yesses";
+  private static final String MAYBES = "maybes";
+  private static final String NOS = "nos";
 
   private Topic goal;
   private MeetingFrequency desiredMeetingFrequency;
+  private Collection<Long> yesses;
```

Comment:
yesses, maybes and nos is a little hard to understand what exactly this collection is storing can we come up with better naming?